dmoj/checkers/standard.py

```diff
@@ -1,10 +1,10 @@
 def check(process_output, judge_output, **kwargs):
```

Comment:
Two bugs here:

* fallback checker does not pass testsuite (fails case 7)
* native checker fails on import
```>>> from dmoj.checkers._checker import standard
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: /mnt/c/dev/judge/dmoj/checkers/_checker.cpython-35m-x86_64-linux-gnu.so: undefined symbol: PyString_Check
```

Perhaps we should just fix the native path and remove the fallback... if you can't compile the checker you can't compile cptbox, and then you're screwed anyway.